subjects/uninvited/README.md

```diff
@@ -0,0 +1,42 @@
+## uninvited
+
+### Instructions
+
+When you started to organize the party you thought it would be easier. Your friend who started helping on the last exercise, raised a question that you didn't think about before. What would happen if people showed up with a plus-one? Or a plus-three?
+
+Oh no! You didn't take into account people uninvited who might come with your guests.
+
+For now, what your friend suggested is to call the guests and try to find out who would come with company.
+
+Create an `uninvited.mjs` program that will open a server to remotely not just access, but also update the list. It will need to handle http `POST` requests to add new guests.
+
+Here below are your program/server's expected behaviors:
+
+- It has to listen on port `5000` and it will have to print a simple message on the console, specifying the listening port.
+- Its HTTP response should contain a coherent status code depending on the handling of the received HTTP request. More specifically, your server should be able to respond with the following status codes: `201` and `500`.
+- The responses will always be JSON and this information should be explicit in the HTTP response.
+- For each http `POST` request, your program should create the corresponding JSON file and store the contents of the body, and then provide the content as JSON in the HTTP response, if possible. If the file already exists, it should replace it.
+- If for any reason the server fails, the response should be an object with a key `error` and its value `server failed`.
+
+### Example
+
+To test your program, you should be able to expect the following behaviour once your program is up and running.
```

Comment:
small typo on "behavior"

Reply:
actually it's not a typo, just UK English... I've checked in the curriculum, there is both versions, US `behavior` and UK `behaviour`, in our subjects.
I have to say we have more times the US version than the UK, I suggest we use that one, but I have no strong opinion so feel free to chose the path you prefer about it.